phosphor-auditlog/meson.build

```diff
@@ -0,0 +1,34 @@
+auditd_dep = dependency('audit')
+ausparse_dep = cpp.find_library('libauparse', required: true)
+
+auditlog_sources = [
+        files(
+        'main.cpp',
+        'auditLogMgr.cpp',
```

Comment:
all of the other files in the repository are snake_case, so I guess it'd be better to do that with these as well.

Reply:
I will make this change to align with the style of this repo.

To make re-review easier I will wait to change the file names until after I've addressed all of the other review comments and you've had a chance to review the updates.

Reply:
Renamed files to follow this style.